### PR TITLE
feat(TDS-2681): add wordComplies operator to TQL to distinguish patte…

### PR DESCRIPTION
**Pull request by gmzoughi**
…rn types.

**What is the problem this Pull Request is trying to solve?**
 
**What is the chosen solution to this problem?**
 
**Link to the JIRA issue**

 
**Please check if the Pull Request fulfills these requirements**
- [ ] The PR commit message follows our [guidelines](../CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features, coverage should be over 75% in the new code)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in JIRA), if any, are all linked or available in the Pull Request

 
**[ ] This Pull Request introduces a breaking change**

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/24/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/24/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/25/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/25/)

File: daikon-tql/daikon-tql-client/README.md [hunk 1]
```diff
@@ -8,8 +8,10 @@ An Operator inherits from the `Operator` class (which implements the `ISerializa
 
 The following operators are supported :
 
-- `complies`
 - `contains`
+- `cotainsIgnoreCase`
```
Comment (scorreia) [inline daikon-tql/daikon-tql-client/README.md]: should read: "containsIgnoreCase"

Comment (gmzoughi) [reply daikon-tql/daikon-tql-client/README.md]: yes!

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/34/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/34/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/35/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/35/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/38/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/38/)

**Pull request merged**